$MirOS: ports/emulators/qemu/patches/patch-block-vvfat_c,v 1.1 2005/11/15 12:25:08 tg Exp $
--- block-vvfat.c.orig	Tue Nov 15 11:52:08 2005
+++ block-vvfat.c	Tue Nov 15 11:52:45 2005
@@ -1737,6 +1737,8 @@ BlockDriver bdrv_vvfat = {
     vvfat_read,
     vvfat_write,
     vvfat_close,
+    NULL,
+    NULL,
+    NULL,
+    NULL
 };
-
-
